velox/functions/prestosql/JsonFunctions.h

```diff
@@ -74,4 +74,25 @@ struct JsonArrayLengthFunction {
   }
 };
 
+template <typename T>
+struct JsonArrayContainsFunction {
```

Comment:
In previous functions I also see the line VELOX_DEFINE_FUNCTION_TYPES(T) where the template variable T is used. That is needed to generate some stub code around this function. I'm surprised that something didn't fail without this code, but you should add it to your function also.

Reply:
Thanks for sharing, I had looked at some templated functions in Arithmetic.h where VELOX_DEFINE_FUNCTION_TYPES(T) was not used so I had removed it, added it back.

Reply:
```const arg_type<> ``` requires VELOX_DEFINE_FUNCTION_TYPES(T), however using this const type for input parameters throws the same UDF error:
```In template: static_assert failed due to requirement 'udf_has_call || udf_has_callNullable || udf_has_callNullFree' "UDF must implement at least one of `call`, `callNullable`, or `callNullFree`".```
Shall I retain this line?